### ISSUE TITLE
UserTracking tab missing?

### ISSUE THREAD

**Issue by Soleg06** [persandstrom/python-verisure#124]
Up until recentely when I called the get_overview() command I got a "userTracing" tab in the json reply but now its missing. Anybody knows if Verisure hidden it or if the information is avalible somevhere else=

**Comment by petertufvesson** [comment]
I also miss this...

**Comment by persandstrom** [comment]
Sorry for a very late reply!
The current implementation is using an old API, which means that I am no longer able to find out how to add new or find modified commands. 
I am working on a new version, which is still under development in the ```m-api``` branch, where user tracking is present. You may use that one but it is not in a stable state yet and lack support for other things, such as camera, mouse, water detector and surely a few other things.

**Comment by krambriw** [comment]
Hello Per,
Been using your verisure sw successfully for several years now. Does the new m-api bring  any particular benefits in comparison? Like does it support push messaging or subscriptions to events?

**Comment by persandstrom** [comment]
No, no big benefits for the end user like push messaging or events. It offers a lot more flexibility in what information to request from the servers, the API is build using GraphQL if you are familiar with that technology.
